README.md

```diff
@@ -35,6 +35,14 @@ Run this to format non-test scala files: `sbt scalafmt`
 
 Run this to format test scala files: `sbt test:scalafmt`
 
+### SBT Alias Commands
+
+Run this to check all code coverage and get a report: `sbt coverageCheck`
```

Comment:
this does more than just check code coverage it also runs all the unit tests and the integration tests - could you update the name and description to make this clear?